examples/ios/swift/AppleAuthentication/README.md

```diff
@@ -1,8 +1,8 @@
-#  How to enable Apple Authentication via MongoDB Realm Cloud
+#  How to enable Apple Authentication via Atlas Device Sync
 
-* You first need to setup your MongoDB Realm app in the cloud. Follow these instructions to get started: https://docs.mongodb.com/realm/authentication/apple/
+* You first need to setup your app in Atlas App Services. Follow these instructions to get started: https://www.mongodb.com/docs/atlas/app-services/authentication/apple/
 * Once that is done add Apple Sign In to your app's entitlements
-* Make sure your bundle ID matches the client id you used in MongoDB Realm Cloud
+* Make sure your bundle ID matches the client id you used in Atlas App Services app.
```

Comment:
https://www.mongodb.com/docs/atlas/app-services/authentication/apple/

Reply:
It's referenced above